automated/linux/uart-loopback/uart-loopback.yaml

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019 Schneider Electric
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: UART Loopback
+    description: "...
+                     ..."
+    maintainer:
+        - ryan.harkin@linaro.org
+    os:
+        - debian
+        - ubuntu
+        - fedora
+        - centos
+        - openembedded
+    scope:
+        - uart
+    devices:
+        - rzn1d
+        - soca9
+
+params:
+        UART0: "/dev/ttyUSB0"
+        UART1: "/dev/ttyS1"
+        YKUSHPORT: 1
+        SKIP_INSTALL: "True"
```

Comment:
I think that SKIP_INSTALL normally means that we skip to install the tests and its dependencies.

Reply:
+1 this is the convention with LTP for example. So this means that build_loopback should be placed inside install(). As a side effect Makefile and uart_loopback.c should be moved outside of this PR (and possibly repository. 

Alternative solution is providing pre-built binaries and storing them in the repository. This is done in case of some benchmarks. It requires to have at least x86 and arm versions of binaries. IMHO this option is simpler and I would prefer it.